nncf/config/schemata/algo/quantization.py

```diff
@@ -526,7 +526,7 @@
         },
         "export_to_onnx_standard_ops": with_attributes(
             BOOLEAN,
-            description="Determines how should the additional quantization "
+            description="(Deprecated) Determines how should the additional quantization "
```

Comment:
Use brackets instead of parentheses as for "disable_shape_matching" so that the deprecation is visible in the visualization of the JSON schema, as in:
https://openvinotoolkit.github.io/nncf/schema/#disable_shape_matching

See also:
https://pypi.org/project/json-schema-for-humans/0.3.1/
(option --deprecated-from-description)

Reply:
Updated